lib/waffle/behaviors/storage_behaviour.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Waffle.StorageBehavior do
+  @callback put(definition :: atom, version :: atom, file_and_scope :: {Waffle.File.t(), term}) :: {:ok, file_name :: String.t()} | {:error, reason :: term}
```

Comment:
Can we use `any` here instead of `term`?

Reply:
Sure. No problem. I fixed it.